mainProject/services/userService.ts

```diff
@@ -0,0 +1,71 @@
+import { IUserRequestBody } from "../interfaces/user";
+import { v4 } from "uuid";
+import UserModel from "../models/user";
+import { Op } from "sequelize";
+
+class UserService {
+  async getAutoSuggestUsers(login: string, limit: number) {
+    const result = await UserModel.findAll({
+      where: {
+        is_deleted: false,
+        login: {
+          [Op.like]: `${login}%`,
+        },
+      },
+      limit,
+    });
+
+    return result.sort((firstObj, secondObj) => {
+      if (firstObj.login > secondObj.login) return 1;
+      if (firstObj.login < secondObj.login) return -1;
+      return 0;
+    });
+  }
+
+  async createUser(userData: IUserRequestBody) {
+    return UserModel.create({
+      id: v4(),
```

Comment:
generating of id could be done automatically by sequilize https://stackoverflow.com/questions/50414899/node-js-sequelize-uuid-primary-key-postgres

Reply:
I knew about it, but it was used on purpose because if I remove the v4() call it means that this package is useless